README.rst

```diff
@@ -1,17 +1,20 @@
-# celery-unique
+celery-unique
```

Comment:
As-discussed, perhaps we can use `pandoc` to assist with automatically converting `README.md` to `README.rst` as a deployment step.  If that seems feasible, we should change this file back to markdown.